app/views/internal/shared/_navbar.html.erb

```diff
@@ -1,9 +1,34 @@
-<% menu_items = %w[comments articles users tags welcome broadcasts reports response_templates pages tools chat_channels permissions growth mods config events badges organizations sponsorships podcasts negative_reactions moderator_actions].sort.each_with_object({}) { |v, h| h[v] = v } %>
-<% menu_items[:listings] = "listings" %>
-<% menu_items[:webhooks] = "webhook_endpoints" %>
+<% menu_items = %w[
```

Comment:
maybe we should freeze this list in the `Internal::ApplicationController` once and for all :D

Reply:
Good idea - done!